frame/nomination-pools/src/lib.rs

```diff
@@ -289,7 +289,7 @@ use sp_staking::{EraIndex, OnStakerSlash, StakingInterface};
 use sp_std::{collections::btree_map::BTreeMap, fmt::Debug, ops::Div, vec::Vec};
 
 /// The log target of this pallet.
-pub const LOG_TARGET: &'static str = "runtime::nomination-pools";
+pub const LOG_TARGET: &str = "runtime::nomination-pools";
```

Comment:
wonder if this str will be inlined as const everywhere it is used, and possible result few copes.

Reply:
It will point into the readonly `.data` section of the executable which has the static strings in it. I would expect same strings to be interned at some point along the way.

Reply:
the documentation says `const` is inlined in every place it's used, and might result to several copies. where `static` item (not lifetime) guarantees a single instance. 
I wonder why we use const in such cases if its intentional. 

https://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/book/first-edition/const-and-static.html#const-and-static

Reply:
if you look at godbolt you can see that it just stores the string once in the binary even with optimisations not turned on: https://godbolt.org/z/3MnW6Tsv3